cmake/templates/pkgConfig.cmake.in

```diff
@@ -117,15 +126,25 @@ foreach(library ${libraries})
   elseif(TARGET ${library})
     list(APPEND @PROJECT_NAME@_LIBRARIES ${library})
   elseif(IS_ABSOLUTE ${library})
-    list(APPEND @PROJECT_NAME@_LIBRARIES ${library})
+    get_filename_component(path ${library} DIRECTORY)
+    get_filename_component(library ${library} NAME)
+    set(lib "lib-NOTFOUND")
+    find_library(lib ${library}
+      PATHS ${path}
+      NO_DEFAULT_PATH)
+    if(NOT lib)
+      message(FATAL_ERROR "Project '${PROJECT_NAME}' tried to find library '${library}'.  The library is neither a target nor built/installed properly.  Did you compile project '@PROJECT_NAME@'?  Did you find_package() it before the subdirectory containing its code is included?")
+    endif()
+    _list_append_unique(@PROJECT_NAME@_LIBRARY_DIRS ${path})
+    list(APPEND @PROJECT_NAME@_LIBRARIES ${lib})
   else()
     set(lib_path "")
-    set(lib "${library}-NOTFOUND")
+    set(lib "lib-NOTFOUND")
```

Comment:
Yes, this is unrelated. However, according to the doc the important part is the `-NOTFOUND` suffix and this is just a cosmetic fix.